includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1181,10 +1181,24 @@ protected function request( $params, $api, $method, $is_site_specific = true, $u
 	 */
 	private function request_with_level3_data( $params, $api, $method, $is_site_specific = true ) {
 		// If level3 data is not present for some reason, simply proceed normally.
-		if ( ! isset( $params['level3'] ) ) {
+		if ( ! isset( $params['level3'] ) || ! is_array( $params['level3'] ) ) {
 			return $this->request( $params, $api, $method, $is_site_specific );
 		}
 
+		// If level3 data doesn't contain any items, add a zero priced fee to meet Stripe's requirement.
+		if ( ! isset( $params['level3']['line_items'] ) || ! is_array( $params['level3']['line_items'] ) || 0 === count( $params['level3']['line_items'] ) ) {
+			$params['level3']['line_items'] = [
+				[
+					'discount_amount'     => 0,
+					'product_code'        => '0-00-fee',
+					'product_description' => '$0.00 fee',
```

Comment:
I think since we're moving forward with other currencies, it might be better to update the code to `zero-cost-fee` and the description to `Zero cost fee`.

Reply:
Sure, I'll change that.